tests/rkt_tests.go

```diff
@@ -297,11 +310,28 @@ func createTempDirOrPanic(dirName string) string {
 	return tmpDir
 }
 
-func importImageAndFetchHash(t *testing.T, ctx *rktRunCtx, img string) string {
+func importImageAndFetchHashAsGid(t *testing.T, ctx *rktRunCtx, img string, gid int) string {
 	// Import the test image into store manually.
-	child, err := gexpect.Spawn(fmt.Sprintf("%s --insecure-skip-verify fetch %s", ctx.cmd(), img))
+	cmd := fmt.Sprintf("%s --insecure-skip-verify fetch %s", ctx.cmd(), img)
+
+	// TODO(jonboulle): non-root user breaks trying to read root-written
+	// config directories. Should be a better way to approach this. Should
+	// config directories be readable by the rkt group too?
```

Comment:
I think it should be readable by the rkt group. If we add the `--auth` option mentioned in #1568 we can leave `/etc/rkt/auth.d` as a place where shared credentials live and use `--auth` for per-user credentials.

We can leave this here in the meantime.